deps/rabbit/src/rabbit_db_cluster.erl

```diff
@@ -306,6 +307,19 @@ members_using_khepri() ->
     %% so we need to allow callers to be more defensive in this case.
     rabbit_khepri:locally_known_nodes().
 
+-spec consistent_members() -> Members when
+      Members :: [node()].
+%% @doc Returns the list of cluster members.
+
+consistent_members() ->
```

Comment:
Can we please add this function to `rabbit_nodes`, for consistency? (it can delegate to this module, just like some [similar functions already do](https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbit/src/rabbit_nodes.erl#L176-L183))

Reply:
I agree, let’s always use `rabbit_nodes` as the public API for cluster members.